src/components/icon/index.jsx

```diff
@@ -3,51 +3,29 @@ import PropTypes from 'prop-types'
 
 import { icon as iconClassName } from './index.css'
 
-import DownloadIcon from './assets/download.svg'
-import OutlineIcon from './assets/outline.svg'
-import ThumbnailsIcon from './assets/thumbnails.svg'
-import PaperInfoIcon from './assets/paper_info.svg'
-import PrintIcon from './assets/print.svg'
```

Comment:
It's a matter of taste but I like to have distinguished files. When new icons are added it will be hard to find index.js file. Just a note. No need to change it. 🙂

Reply:
It's better to have less changes. If you add icon, you need only to place a file and everything works. If you have imports, you have to add a file and update JS. It may have significant impact of mistake at some cases.

Reply:
Yeah I agree